db_scripts/db_dump

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+NAME=${1:-postgres}
```

Comment:
I think default search name should be `wubloader_postgres`. I think that's what docker-compose calls it.

Reply:
No, the default is directoryname_postgres_1.